.github/workflows/markdownlint.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
* this removes `node` as a dependency for running pre-commit 
* this action is maintained by the same maintainer as markdownlint itself